frontend/src/Components/LoginForm/LoginForm.css

```diff
@@ -0,0 +1,85 @@
+.Starter {
+    width: 420px;
+    background: #7692ff;
+    border-radius: 10px;
+    padding: 30px 40px;
+}
+
+.Starter h1 {
+    font-size: 36px;
+    text-align: center;
+}
+
+.Starter .input-box {
+    position: relative;
+    width: 100%;
+    height: 50px;
+    margin: 30px 0;
+}
+.input-box input {
+    width: 100%;
+    height: 100%;
+    background: transparent;
+    border: none;
+    outline: none;
+    border: 2px solid rgba(255, 255, 255, .2);
+    border-radius: 40px;
+    font-size: 16px;
+    color: #000;
+    padding: 20px 45px 20px 20px;
+}
+
+.input-box input::placeholder {
+    color: #fff;
+}
+
+.Starter .remember-forgot {
+    display: flex;
+    justify-content: space-between;
+    font-size: 14.5px;
```

Comment:
Okay, I'll try and edit it accordingly!

Reply:
actually this is not always the case. font size can def be a `px` based property, since you don't necessarily want the font size to scale directly with the size of your browser